src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -94,23 +127,38 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime is recommended to use for storing date values in a database.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return localTime.atOffset(ZoneOffset.of("+02:00"));
     }
 
     /**
      * Given a String formatted as `yyyymmdd`,
      * return Optional of this date as a LocalDate.
      */
     public Optional<LocalDate> parseDate(String date) {
-        return Optional.empty();
+        try {
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyyMMdd");
```

Comment:
there is a specific BASIC_ISO_DATE formatter in DateTimeFormatter

Reply:
agree